libs/features/game-tele/src/edit-game-tele/game-tele.integration.spec.ts

```diff
@@ -0,0 +1,156 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { RouterTestingModule } from '@angular/router/testing';
+import { GameTele } from '@keira/shared/acore-world-model';
+import { MysqlQueryService, SqliteService } from '@keira/shared/db-layer';
+import { EditorPageObject, TranslateTestingModule } from '@keira/shared/test-utils';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { ToastrModule } from 'ngx-toastr';
+import { of } from 'rxjs';
+import { instance, mock } from 'ts-mockito';
+import { GameTeleComponent } from './game-tele.component';
+import Spy = jasmine.Spy;
+import { GameTeleHandlerService } from '../game-tele-handler.service';
+
+class GameTelePage extends EditorPageObject<GameTeleComponent> {}
+
+describe('GameTele integration tests', () => {
+  let fixture: ComponentFixture<GameTeleComponent>;
+  let queryService: MysqlQueryService;
+  let querySpy: Spy;
+  let handlerService: GameTeleHandlerService;
+  let page: GameTelePage;
```

Comment:
this is not critical, and unfortunately, we still have shared variables here and there, but it would be better to avoid having shared non-immutable variables across different tests and just initialise and return them in the `setup` function. See https://francescoborzi.github.io/ngx-page-object-model/docs/best-practices/avoid-shared-variables